### DISCUSSION TITLE
How to identify new JA3/JA3S fingerprint pairs?

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#4848] [Unsupported Versions] [answered]
Hello!

I get a lot of JA3 related false positives from Suricata rules and would be much more interested in the following: How can I identify new (that I have never seen before in my network) pairs of JA3 and JA3S fingerprints that have been identified by zeek and stored in Elasticsearch? Is that something for a Play in Playbook? Can I archive that with Elastalert? Any great ideas out there? Thanks in advance!

Chers, Ben

**Comment by defensivedepth** [comment, marked as answer]
You may be able to do something like that via a `New Term` Elastalert rule: https://elastalert.readthedocs.io/en/latest/ruletypes.html#new-term
